MycroftAI/mycroft-core
# Upgrade websocket-client to 0.58.0

## Description
This is the core equivalent of https://github.com/MycroftAI/mycroft-messagebus-client/pull/21
There was an incompability with the latest websocket-client and the
messagebus which needed fixing for Linux distro compatibility. Since
messagebus-client was going to depend on websocket-client 0.58.0, let's
do the same in core

## How to test
Run this combined with https://github.com/MycroftAI/mycroft-messagebus-client/pull/21 and an upgraded websocket-client, see if everything still works as expected.

## Contributor license agreement signed?
CLA [x]
